.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Is this a vscode setup file? It looks like we're ignoring `**/.vscode` in the `.gitignore` but this file still made it in - could we remove this?

Reply:
Yes it is - it slipped through - i will remove in the next commit.